addon/components/api/x-section/index.js

```diff
@@ -2,16 +2,14 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 import config from 'ember-get-config';
 
-const { showImportPaths } = config['ember-cli-addon-docs'];
-
 /**
   @class Api/XSection
   @hide
 */
-export default Component.extend({
-  tagName: '',
+export default class ApiXSection extends Component {
+  tagName = '';
 
-  showImportPaths,
+  showImportPaths = config['ember-cli-addon-docs'];
```

Comment:
This does not seem correct. `showImportPaths` was destructured before.

Reply:
@rwwagner90 ah yes, good catch!
Actually I'll revert this change, I planned to make native class conversion in separate PR to make things focused

Reply:
reverted!